libs/cpp.json

```diff
@@ -319,13 +319,11 @@
   {"id":"cpp.cpr","name":"C++ Requests","tech":["internet"],"tags":["curl"],"repo":"whoshuu/cpr","imports":["cpr/cpr.h"],"examples":["https://github.com/whoshuu/cpr/tree/master/test"],"status":"model-ready"},
   {"id":"cpp.curlcpp","name":"curlcpp","tech":["internet"],"tags":["curl-wrapper"],"repo":"JosephP91/curlcpp","imports":["curlcpp/"],"examples":["https://github.com/JosephP91/curlcpp#simple-usage-example"],"status":"model-ready"},
   {"id":"cpp.http-parser","name":"HTTP Parser","tech":["internet"],"tags":["http-parser"],"repo":"nodejs/http-parser","imports":["http_parser.h"],"examples":["https://github.com/nodejs/http-parser/blob/master/test.c"],"status":"model-ready"},
-  {"id":"cpp.libcurl","imports":["curl/curl.h"],"name":"libcurl","repo":"curl/curl","tags":["curl"],"tech":["internet"],"status":"awaiting-model"},
   {"id":"cpp.libcurl","name":"libcurl","tech":["internet"],"tags":["curl"],"repo":"curl/curl","imports":["curl/"],"examples":["https://curl.haxx.se/libcurl/c/allexamples.zip"],"status":"model-ready"},
   {"id":"cpp.libtorrent","name":"libtorrent","tech":["internet"],"tags":["bittorrent"],"repo":"arvidn/libtorrent","imports":["libtorrent/"],"examples":["https://github.com/arvidn/libtorrent/blob/master/examples"],"status":"model-ready"},
   {"id":"cpp.libutp","name":"libutp","tech":["internet"],"tags":["bittorrent"],"repo":"bittorrent/libutp","imports":["utp.h"],"examples":["https://github.com/bittorrent/libutp/blob/master/ucat.c"],"status":"model-ready"},
   {"id":"cpp.mongoose","name":"mongoose","tech":["internet"],"tags":["web-server"],"repo":"cesanta/mongoose","imports":["mongoose.h"],"examples":["https://github.com/cesanta/mongoose/tree/master/examples"],"status":"model-ready"},
   {"id":"cpp.nghttp2","name":"nghttp2","tech":["internet"],"tags":["http2"],"repo":"nghttp2/nghttp2","imports":["nghttp2/"],"examples":["https://github.com/nghttp2/nghttp2/tree/master/examples"],"status":"model-ready"},
-  {"id":"cpp.onion","name":"Onion","tech":["internet"],"tags":["http-server"],"repo":"davidmoreno/onion","imports":["onion/"],"examples":["https://github.com/davidmoreno/onion/tree/master/examples"],"status":"model-ready"},
```

Comment:
why did you delete onion? does not seem to be a duplicate.

Reply:
it has 2nd cpp.onion, just of different tech